ui/src/vayla-design-lib/menu/menu.tsx

```diff
@@ -94,9 +78,11 @@ export const Menu = function <TValue>({
                                     i.disabled && styles['menu__item--disabled'],
                                 )}
                                 onClick={() => {
-                                    if (!i.disabled) {
-                                        if (i.type === 'SELECT') i.onSelect();
-                                        else if (onSelect) onSelect(i.value);
+                                    if (!i.disabled && i.type === 'SELECT') {
+                                        i.onSelect();
+                                        if (i.closeAfterSelect && onClose) {
+                                            onClose();
+                                        }
```

Comment:
More 🥩

Reply:
Tuo `i` on yleensä indeksiä kuvaava nimi muuttujalle (ja piti tarkistaa että mistä se on oikeasti peräisin), olisin varmaan itse pitänyt tuon esim. pidempänä ja kuvaavampana `item`inä

Reply:
Wanhaa perua tuo `i`, enkä huomannut tässä kyseenalaistaa. Mutta joo, voisihan sen nimetä paremminkin